### ISSUE TITLE
Loss in fastflow

### ISSUE THREAD

**Issue by ataylor-mw** [RistoranteRist/FastFlow#3]
https://github.com/RistoranteRist/FastFlow/blob/86396fe72ba172d4ab6257a56bff1bc5994b1e0e/utils.py#L8

In the paper in the introduction section they describe in the contributions:

"We propose a 2D normalizing flow denoted as FastFlow
for anomaly detection and localization with fully convolutional networks and two-dimensional loss function to
effectively model global and local distribution"

Is there a reason why the loss isn't computed pixelwise prior to mean reduction. Right now you mean reduce both the Z.^2 and J terms separately prior to computing the difference. From the phrasing of the paper, i would have expected this to be written as:

```
def calc_loss(z,j):
    return torch.sum(0.5*z**-j,(1,2,3))
```
 
Obviously I'm guessing since the paper doesn't describe specifics, but just thought I'd check how you arrived at the current form of the loss?
